packages/cli/src/actions/create.ts

```diff
@@ -228,7 +227,7 @@ console.log(prefix, chalk.green("Presence created! You can now start coding!"));
 
 async function serviceExists(service: string) {
 	try {
-		await access(`./websites/${getFolderLetter(service)}/${service}`);
+		await access(`./websites/${service.at(0)!.toUpperCase()}/${service}`);
```

Comment:
Why is it not using getFolderLetter?

Reply:
Code is based of the previous release. So it was never used in there.
Will update and test tomorrow.